main.py

```diff
@@ -0,0 +1,146 @@
+import json
+
+
+def login() -> int:
+
+    """
+    This function asks the user for his customer ID and password
+    return: customer ID when the password is correct
+    """
+
+    target_id = int(input("Enter your customer ID"))
+    with open("customers.json", 'r') as customers_file:
+        customers_data = json.load(customers_file)
+        list_of_costumers_dicts = customers_data["customers"]
+        for customer in list_of_costumers_dicts:
+            if customer["customer_id"] == target_id:
+                password = int(input("Enter your password"))

```

Comment:
I'd change password into a string, to make sure it has more options (so it's less vulnerable to dictionary attacks) and more memorable